java/ql/src/Security/CWE/CWE-502/UnsafeDeserialization.ql

```diff
@@ -54,6 +66,33 @@ class UnsafeDeserializationConfig extends TaintTracking::Configuration {
       ma.getArgument(0) = node.asExpr() and
       exists(SafeJsonIoConfig sji | sji.hasFlowToExpr(ma.getArgument(1)))
     )
+    or
+    exists(MethodAccess ma |
+      (
+        ma.getMethod() instanceof FlexjsonDeserializeMethod or
+        ma.getMethod() instanceof JoddJsonParseMethod
+      ) and
+      node.asExpr() = ma.getArgument(0) and
+      (
+        ma.getArgument(1)
+            .(TypeLiteral)
+            .getType()
+            .(ParameterizedType)
+            .getATypeArgument()
+            .(Class)
+            .isFinal()
```

Comment:
Give an example of what this means as a comment, especially why being final is significant

Reply:
Thanks @smowton for the review. I've added an example.